www/redirects.yaml

```diff
@@ -138,3 +138,7 @@
   # redirecting gatsby-source-filesystem Programmatic Import to plugin README
 - fromPath: /docs/gatsby-source-filesystem-programmatic-import/
   toPath: /packages/gatsby-source-filesystem/
+- fromPath: /contributing/blog-and-website-contributions/
```

Comment:
one more offtopic: i would only rename `toPath` to `gotoPath` so that all entries are inline :)

```yaml
- fromPath: /contributing/docs-and-website-components/
  gotoPath: /contributing/docs-and-blog-components/
```